mariadb-cluster/manifests/mariadb-master-rc.yaml

```diff
@@ -0,0 +1,68 @@
+#helm:generate helm tpl -d tpl/values.toml -o manifests/mariadb-master-rc.yaml $HELM_GENERATE_FILE
```

Comment:
We could remove the line but I would prefer that if the user executes `helm generate` then the identical file is generated.